x/ibc/04-channel/keeper/timeout.go

```diff
@@ -91,6 +86,11 @@ func (k Keeper) TimeoutPacket(
 
 	switch channel.Ordering {
 	case exported.ORDERED:
+		// check that packet has not been received
```

Comment:
This check should only be performed for ORDERED channels

Reply:
This is a mistake in the spec, sorry - fixed in https://github.com/cosmos/ics/commit/a819bb56f28a2b2678ab1ec02f417bfb8c93565b